requirements.txt

```diff
@@ -5,3 +5,5 @@ mongogrant==0.2.2
 mongomock==3.16.0
 pymongo==3.8.0
 django==2.2.5
+elasticsearch_dsl==6.4.0
+ase==3.15.0
```

Comment:
I'm a bit uncomfortable with adding ASE as dependency given how heavyweight it is. Can we just include a text file with element symbol and atomic number for this purpose? A more general issue is whether we should split the requirements further to have e.g. mongo_requirements.txt, django_requirements.txt etc. Would you be able to do that in this PR too?

Reply:
I removed the ase dependency and just added the chemical symbol array instead. I also followed you example and put the elastic specific requirements in an elastic_requirements.txt.